source/game.c

```diff
@@ -310,6 +320,10 @@ int get_played_top(void) {
     return played_top;
 }
 
+int get_scored_card_index(void) {
+    return scored_card_index;
+}
```

Comment:
```suggestion
int get_scored_card_index(void) 
{
    return scored_card_index;
}
```

Use Allman indentation style